resources/strings.xml

```diff
@@ -1,6 +1,9 @@
 <strings>
     <string id="AppName">JSON Face</string>
     <string id="title">time.json</string>
+
+    <!-- VALUES -->
+    <string id="empty">----</string>
```

Comment:
I would place this string below settings since ir is only used for this purpose

Reply:
Reordered